assignment1/src/main/java/ro/utcn/sd/he/assignment1/model/Post.java

```diff
@@ -0,0 +1,17 @@
+package ro.utcn.sd.he.assignment1.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.sql.Timestamp;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public abstract class Post {
+    private int id;
+    private String author;
+    private String text;
+    private Timestamp creation_date_time;
```

Comment:
For future reference: check out the Java 8 newer types like `LocalDateTime` as a replacement for the older `Timestamp` and `Date` classes.

Reply:
I used timestamp because i found that it was compatible with the MySQL datatype. I'll note this and try to use the newer ones from here on. Thanks.